### ISSUE TITLE
High dimensional McKean Vlasov FBSDE using NN

### ISSUE THREAD

**Issue by finmod** [EmoryMLIP/MFGnet.jl#2]
Can MFGnet add this example raised by  https://arxiv.org/pdf/1909.12678.pdf  . It would be a nice addition to Julia. See also: https://github.com/JuliaDiffEq/NeuralNetDiffEq.jl/issues/52

**Comment by lruthotto** [comment]
Yes, in principle this is a natural extension, but manpower could be an issue.
